src/networking/httpService.ts

```diff
@@ -13,6 +13,75 @@ async function get<T = any>(endpoint: string): Promise<T> {
   }
 }
 
+async function post<T = any>(
+  endpoint: string,
+  title: string,
+  description: string,
+): Promise<T> {
+  try {
+    const response = await fetch(baseUrl + endpoint, {
+      body: JSON.stringify({
+        description: description,
+        title: title,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      method: 'POST',
+    })
+    if (!response.ok) {
+      throw new Error('The new todo could not be uploaded')
+    }
+    const data = await response.json()
+    return data as T
+  } catch (error) {
+    throw new Error(error.message)
+  }
+}
+
+async function put<T = any>(
+  endpoint: string,
+  title: string,
+  description: string,
+  toCheck: boolean,
+): Promise<T> {
+  try {
+    const response = await fetch(baseUrl + endpoint, {
+      body: JSON.stringify({
+        completed: toCheck,
+        description: description,
+        title: title,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      method: 'PUT',
+    })
+    if (!response.ok) {
+      throw new Error('') //No error message so todos continue appearing, problem that error is not shown
```

Comment:
Remember to add the prefix "`// TODO: `" in each comment that has something pending to resolve.

Can you explain what is the expected behaviour with this one?